modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -856,30 +895,34 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeStressFinalize(
 
   _cell_rate[_qp] = _cell_dislocation_increment / _dt;
   _wall_rate[_qp] = _wall_dislocation_increment / _dt;
+
   _cell_dislocations[_qp] = _old_input_values[_cell_output_index] + _cell_dislocation_increment;
   _wall_dislocations[_qp] = _old_input_values[_wall_output_index] + _wall_dislocation_increment;
 
+  // For (possibly) substepping.
+  _plastic_strain_increment += MetaPhysicL::raw_value(plastic_strain_increment);
+
   // Prevent the ROM from calculating and proceeding with negative dislocations
   if ((_cell_dislocations[_qp] < 0.0 || _wall_dislocations[_qp] < 0.0) && (this->_apply_strain))
   {
     const Real negative_cell_dislocations = MetaPhysicL::raw_value(_cell_dislocations[_qp]);
     const Real negative_wall_dislocations = MetaPhysicL::raw_value(_wall_dislocations[_qp]);
     _cell_dislocations[_qp] = _old_input_values[_cell_output_index];
     _wall_dislocations[_qp] = _old_input_values[_wall_output_index];
-    mooseException("The negative values of the cell dislocation density, ",
-                   negative_cell_dislocations,
-                   ", and/or wall dislocation density, ",
-                   negative_wall_dislocations,
-                   ". Cutting timestep.");
+    mooseError("The negative values of the cell dislocation density, ",
```

Comment:
This will no longer cut the timestep so you can get rid of that part of that part of the error.

Reply:
Maybe this error should be after the verbose output.

Reply:
This was a local change I made to make sure I didn't run into these undesirable situations while testing or creating tests (I pushed this as a draft to trigger CIVET testing). I will revert this back to being an exception to avoid any regression in behavior.